### PR TITLE
Update README.md for upcoming v1.0.0

### PR DESCRIPTION
**Pull request by Nashatyrev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,6 +126,12 @@ To build the library from the `jvm-libp2p` folder, run:
 
 After the build is complete you may find the library `.jar` file here: `jvm-libp2p/build/libs/jvm-libp2p-minimal-0.x.y-RELEASE.jar`
 
+## Notable users
+
+- [Teku](https://github.com/ConsenSys/teku) - Ethereum Consensus Layer client
```
Comment (StefanBratanov) [inline README.md:131]: just a nit, change to Consensys with a lowercase

**Review by StefanBratanov — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -6,58 +6,44 @@
...
+|--------------------------|-------------------------------------------------------------------------------------------------|:------------------------:|
+| **Transport**            | tcp                                                                                             |    :green_apple:    |
+|                          | [quic](https://github.com/libp2p/specs/tree/master/quic)                                        | :tomato: |
+|                          | websocket                                                                                       |    :lemon:     |
+|                          | [webtransport](https://github.com/libp2p/specs/tree/master/webtransport)                        |                       |
+|                          | [webrtc-browser-to-server](https://github.com/libp2p/specs/blob/master/webrtc/webrtc-direct.md) |                       |
+|                          | [webrtc-private-to-private](https://github.com/libp2p/specs/blob/master/webrtc/webrtc.md)       |                       |
+| **Secure Communication** | [noise](https://github.com/libp2p/specs/blob/master/noise/)                                     |    :green_apple:    |
```
Comment (StefanBratanov) [inline README.md:23]: Maybe can add secio in secure communication section? https://github.com/libp2p/specs/blob/master/secio/README.md

Comment (Nashatyrev) [reply README.md:23]: I was deprecated ans is not more in the official list in Libp2p spec

Comment (Nashatyrev) [reply README.md:23]: However we may add it with 'deprecated' mark :)

Comment (StefanBratanov) [reply README.md:23]: Ah got it, can ignore it then if deprecated anyways.

**Comment by Nashatyrev** [comment]
Added 'Android support' notes. @StefanBratanov could you please review it (with respect to English language as well) ?

File: README.md [hunk 3]
```diff
@@ -6,65 +6,64 @@
...
 
 ## Gossip simulator
 
 Deterministic Gossip simulator which may simulate networks as large as 10000 of peers
 
 Please check the Simulator [README](tools/simulator/README.md) for more details
 
+## Android support
+
+The library is basically being developed with Android compatibility in mind. However we don't aware of any library usage
```
Comment (StefanBratanov) [inline README.md:57]: `However we are not aware of anyone using it in production` sounds better in my opinion

**Comment by StefanBratanov** [comment]
> Added 'Android support' notes. @StefanBratanov could you please review it (with respect to English language as well) ?

LGTM

**Pull request merged**